av1an-core/src/settings.rs

```diff
@@ -535,6 +535,14 @@ properly into a mkv file. Specify mkvmerge as the concatenation method by settin
       );
     }
 
+    if matches!(self.encoder, Encoder::aom | Encoder::vpx)
+      && self.passes == 2
```

Comment:
I wonder if we want to make this `>= 2` in case some crazy person tries to use 3-pass with `--rt`?

Reply:
I'll do that in case we ever support 3-pass, although av1an will not let you set anything other than 1 or 2 for passes